libs/shared/connected/src/utils/sortAlpha.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * NOTE:
+ *  The contents of this file are the same as in the file apps/web/utils/sortAlpha.ts
+ *  We cannot import from that file, as it will create circular dependency.
+ *  Therefor, at least for now, we copy this code here. Ideally, this code would be
+ *  moved into a dedicated utils library.
+ */
```

Comment:
Should move this to `libs/shared/utils` as a first step.

Though this shared/connected library (and way more "web"-related code) should be in `libs/web/*` instead of under shared, since they won't really be used by any other projects.